zipline-testing/src/appleMain/kotlin/app/cash/zipline/testing/CoroutineDispatchersApple.kt

```diff
@@ -16,13 +16,15 @@
 package app.cash.zipline.testing
 
 import kotlin.coroutines.CoroutineContext
+import kotlinx.cinterop.ExperimentalForeignApi
 import kotlinx.cinterop.convert
 import kotlinx.coroutines.CloseableCoroutineDispatcher
 import kotlinx.coroutines.Runnable
 import kotlinx.coroutines.channels.Channel
 import kotlinx.coroutines.runBlocking
 import platform.Foundation.NSThread
 
+@OptIn(ExperimentalForeignApi::class)
```

Comment:
This actual opts-in, but the expect and the other actuals propagate the annotation. Should they all opt-in instead?

Reply:
It's actually separate experimental annotations: the expects propagate `@ExperimentalCoroutinesApi` as it exposes `CloseableCoroutineDispatcher`, but this opts into `@ExperimentalForeignApi`.